src/adapter-types/src/dyncfgs.rs

```diff
@@ -32,6 +32,12 @@ pub const WITH_0DT_DEPLOYMENT_MAX_WAIT: Config<Duration> = Config::new(
     "How long to wait at most for clusters to be hydrated, when doing a zero-downtime deployment.",
 );
 
+pub const WITH_0DT_DEPLOYMENT_PANIC_AFTER_TIMEOUT: Config<bool> = Config::new(
+    "0dt_panic_after_timeout",
```

Comment:
Intentional that the name `0dt_panic_after_timeout` is different from `WITH_0DT_DEPLOYMENT_PANIC_AFTER_TIMEOUT`? I thought "with" is for durations, while "enable" is for booleans.